Packs/Zoom/Integrations/Zoom_IAM/Zoom_IAM.py

```diff
@@ -0,0 +1,291 @@
+DEFAULT_INCOMING_MAPPER = "User Profile - SCIM (Incoming)"
+
+ERROR_CODES_TO_SKIP = [
+    404
+]
+BASE_URL = 'https://api.zoom.us/v2/'
+
+
+'''CLIENT CLASS'''
+
+
+class Client(BaseClient):
+    """ A client class that implements logic to authenticate with Zoom application. """
+
+    def __init__(self, base_url, api_key, api_secret, verify=True, proxy=False):
+        super().__init__(base_url, verify, proxy)
+        self.api_key = api_key,
+        self.api_secret = api_secret,
+        self.access_token = get_jwt(api_key, api_secret)
+        # self.headers = {'authorization': f'Bearer {self.access_token}',
+        #                 'Accept': 'application/json',
+        #                 'Content-Type': 'application/json'
+        #                 }
+
+    def test(self):
+        """ Tests connectivity with the application. """
+
+        self.get_user('me')
+        return 'ok'
+
+    def get_user(self, email: str) -> Optional[IAMUserAppData]:
+        """ Queries the user in the application using REST API by its email, and returns an IAMUserAppData object
+        that holds the user_id, username, is_active and app_data attributes given in the query response.
+
+        :type email: ``str``
+        :param email: Email address of the user
+
+        :return: An IAMUserAppData object if user exists, None otherwise.
+        :rtype: ``Optional[IAMUserAppData]``
+        """
+        uri = f'/users/{email}'
+
+        res = self._http_request(
+            method='GET',
+            url_suffix=uri,
+            headers={'authorization': f'Bearer {self.access_token}',
+                     'Accept': 'application/json',
+                     'Content-Type': 'application/json'
+                     }
+        )
+        if res and (not res.get('users')):
+            user_app_data = res
+            user_id = user_app_data.get('id')
+            is_active = True if user_app_data.get('status') == 'active' else False
+            username = ''
+
+            return IAMUserAppData(user_id, username, is_active, user_app_data)
+        return None
+
+    def update_user(self, user_id: str, user_data: Dict[str, Any]) -> IAMUserAppData:
+        """ Updates a user in the application using REST API.
+
+        :type user_id: ``str``
+        :param user_id: ID of the user in the application
+
+        :type user_data: ``Dict[str, Any]``
+        :param user_data: User data in the application format
+
+        :return: An IAMUserAppData object that contains the data of the updated user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+        uri = f'/users/{user_id}/status'
+        res = self._http_request(
+            method='PUT',
+            url_suffix=uri,
+            json_data=user_data,
+            headers={'authorization': f'Bearer {self.access_token}',
+                     'Accept': 'application/json',
+                     'Content-Type': 'application/json'
```

Comment:
```suggestion
                     'Content-Type': 'application/json',
```

see https://stackoverflow.com/questions/6388187/what-is-the-proper-way-to-format-a-multi-line-dict-in-python